composer.json

```diff
@@ -8,20 +8,22 @@
         "ext-curl": "*",
         "ext-dom": "*",
         "ext-soap": "*",
+        "ext-xsl": "*",
         "cweagans/composer-patches": "^1",
-        "nyholm/psr7": "^1.4",
-        "php-http/guzzle6-adapter": "^2.0",
         "php-http/logger-plugin": "^1.3",
         "phpro/soap-client": "~2.0.0",
-        "psr/http-factory-implementation": "*",
-        "psr/log": "~1",
+        "psr/http-client-implementation": "^1",
+        "psr/http-factory-implementation": "^1",
+        "psr/log": "^1 || ^3",
```

Comment:
Shouldn't we require `psr/http-client: ^1.0`, now that we don't require `php-http/guzzle6-adapter` anymore?

Reply:
Yes indeed, I missed that. It contains the `ClientInterface` which is in use in this library. Going to add it now.

Reply:
Job done.